src/autoapp/Service/AndroidAutoEntity.cpp

```diff
@@ -247,6 +247,15 @@ void AndroidAutoEntity::triggerQuit()
     }
 }
 
+/*
+ * As of Google Play Services 15.0.90 (Jan 29 2019), this function causes
+ * the Android Auto interface to crash without the ability to cover seconds
+ * after connection. 
+ * 
+ * Simply commenting this function out is a temporary fix to allow Crankshaft
+ * users to use Android Auto with the most Google Play Services >= 15.0.90
+ * but should be properly addressed in the future.
+ *
 void AndroidAutoEntity::schedulePing()
```

Comment:
Shouldn't line 63 on this file be commented instead of this whole function?

Reply:
@htruong Sorry for the late reply. I've referenced lines 250 through 267, are you referring to line 263 when you say line 63?

In either case, I've commented out the entire function because I don't fully understand how OpenAuto and aasdk work together and thought it best to completely remove the function rather than change how it works.

I'm a college student and I am going to a hackathon this weekend where I plan to spend a few days working on this project. Hopefully I can be more helpful after I've spent more time understanding how everything works.